site/_posts/2021-05-11-manageiq-perf-results.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "ManageIQ performance results and improvements"
+author: Thad Jennings
+date: 2021-05-11
+tags: performance
+comments: false
+published: false
+---
+
+
+# ManageIQ performance results and improvements
+
+Thad Jennings (performance engineer)
+
+
+## Introduction
+
+This document decribes performance measurements for the ManageIQ Kasparov release in both podified and appliance mode, and performance improvements that have been made as a result.  The purpose of these measurements was to show the behavior of ManageIQ as the number of providers and managed resources are increased, and to look for inhibitors to scalability.  Podified measurements were taken on Red Hat OpenShift Container Platform (OCP) running on a VMware vCluster dedicated to performance measurements.  Appliance measurements were taken on a VM running on the same VMware vCluster.  
+
+### Measurement configuration
+
+These measurements were taken on a vCenter dedicated to performance measurements with the following configuration:
+* Hypervisor:		VMware ESXi, 6.7.0, 15160138
+* Model:			IBM System X iDataPlex dx360 M4 Server -[7912AC1]-
+* Processor Type:		Intel(R) Xeon(R) CPU E5-2665 0 @ 2.40GHz
+* 14 ESX hosts: 		16 cores (32 logical processors), 256 GB Total Memory
+
+The podified measurements were taken on an OCP cluster with the following characteristics:
+* OCP 4.6.15
+* Kasparov branch
+* 3 OCP masters (8 core, 32 GB RAM)
+* 4 workers (8 core, 32 GB RAM)
+
+The appliance measurements were taken on a single VM running on the same vSphere cluster, with 16 cores, 64 GB RAM and a separate 200 GB hard disk.
+
+ManageIQ was configured with all server roles enabled, except Git Repositories Owner.  C & U Collection (metric collection) was enabled for all clusters and all datastores.
+
+For these measurements, up to 13 providers were added in a single, default zone:
+* 1 Amazon Web Service cloud provider
+* 1 Azure cloud provider
+* 1 VMware infrastructure provider
+* 10 additional VMware infrastructure providers simulated with the vCenter Server Simulator (VCSIM) tool, which is an unsupported tool available for free from VMware.  Each simulated vCenter runs on a separate VM.  
+
+### Steady state CPU utilization
+<img src="/assets/images/blog/PodSteadyStateCPU.png" width="250"></a>

```

Comment:
```suggestion
<img src="/assets/images/blog/PodSteadyStateCPU.png" width="250">
```

The closing anchor tag should be removed because there's no opening anchor tag from what I can tell - I think the other images have the same closing anchor tag.